src/Storefront/Resources/views/storefront/layout/navbar/navbar.html.twig

```diff
@@ -1,10 +1,15 @@
 {% block layout_navbar %}
     <div class="container">
+        {% set navbarOptions = {
+            pathIdList: shopware.navigation.pathIdList
+        } %}
+
         <nav class="navbar navbar-expand-lg main-navigation-menu"
```

Comment:
should we add a block around the `nav` element, so the `navbarOptions` could be adjusted?